shared/fs/container.tsx

```diff
@@ -39,8 +39,8 @@ const ChooseComponent = (props: ChooseComponentProps) => {
 
   Kbfs.useFsPathMetadata(props.path)
   Kbfs.useFsFileContext(props.path)
-  Kbfs.useFsTlfs()
-  Kbfs.useFsOnlineStatus()
+  Container.isMobile && Kbfs.useFsTlfs()
```

Comment:
why only on mobile? / can you add a comment?

Reply:
Because I didn't make it load at start on mobile yet. Wanted to see what people think about it on desktop first, and maybe wait until we have a better solution for loading it at start that wouldn't be as aggressive.